docker/docker-compose.postgres.yml

```diff
@@ -1,13 +1,11 @@
 services:
   website:
     environment:
-      DB_CONNECTION: pgsql
       DB_LOGIN: postgres
       DB_PORT: 5432
   # Uncomment this section for production postgres installations.
   #worker:
```

Comment:
We should test this locally, but I think we can uncomment these lines now, saving our users a manual step in doing so.